docs/building/packaging.md

```diff
@@ -0,0 +1,71 @@
+# Creating Tracee Linux packages
+
+> These instructions are meant to describe how to build Linux distributions
+> packages If you would like to have a local building building and execution
+> environment, [read this](./environment.md) instead.
+
+## Ubuntu Linux
+
+Tracee has a directory "packaging" with a skeleton of the needed "debian/"
+directory called "ubuntu". This skeleton is used by "packaging/ubuntu-pkgs.sh"
```

Comment:
Can you explain what this means? The needed debian directory called ubuntu?

Reply:
Sure. 

This "ubuntu" directory is, indeed, a "src-tree/debian" directory, needed for debian package. The reason for it NOT to be called "debian" is because I'm reserving "debian" for the Debian set of packages.

The contents of this "ubuntu/" directory are the same as the "src/debian" directory of a debian/ubuntu package, but there are slight differences among debian & ubuntu that will be kept separately. 

So, at the end, we may have something like:

./builder/packaging/ubuntu => will be src_tree/debian during package builds

./builder/packaging/debian => will be src_tree/debian during package builds

./builder/packaging/fedora => will contain the RPMBUILD file

./builder/packaging/archlinux => will contain the PKGBUILD file

and so on...